client/redux/catalog/reducers.js

```diff
@@ -0,0 +1,35 @@
+import { FETCH_PRODUCTS_REQUEST, FETCH_PRODUCTS_SUCCESS, FETCH_PRODUCTS_FAILURE } from './constants'
```

Comment:
@AStashevskaya Lets try to combine `catalog actions`, `catalog constants`, and `catalog reducers` in single file for `catalog store`